cloudbuild/nightly.yaml

```diff
@@ -0,0 +1,39 @@
+steps:
+  # 1. Create a Docker image containing flink-bigquery-connector repo
+  - name: 'gcr.io/cloud-builders/docker'
+    id: 'docker-build'
+    args: ['build', '--tag=gcr.io/$PROJECT_ID/dataproc-flink-bigquery-connector-nightly', '-f', 'cloudbuild/Dockerfile', '.']
+
+# 2. Fetch maven and dependencies
```

Comment:
Indentation seems off compared to line 2

Reply:
Fixed.